### PR TITLE
support dkg command

### PR DESCRIPTION
**Pull request by corverroos**
Adds support for `make dkg` to simulate local DKG.

### CODE REVIEW

**Review by dB2510 — APPROVED** [review]
LGTM

File: docker-compose.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ x-node-base: &node-base
   # Pegged charon version (update this for each release).
   image: ghcr.io/obolnetwork/charon:v0.3.0
   entrypoint: "${COMPOSE_ENTRYPOINT:-/usr/local/bin/charon}"
-  command: run
+  command: "${COMPOSE_COMMAND:-run}"
```
[the remark this answers is not in the document]
Comment (corverroos) [reply docker-compose.yml:7]: It allows overriding the command (to run dkg instead), and sets the default to `run` if it isn't overridden.

File: configure_dkg.sh [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+for i in $( eval echo {0..$N} ); do
+  [ "${i}" == "$N" ] && break
+
+  ENR=$($CMD gen-p2pkey --data-dir=/charon-docker-compose/node${i} | grep enr)
+  [ "${i}" != "0" ] && ENRS+=","
+  ENRS+=$ENR
+done
+
+echo "Creating cluster_definition.json file"
+$CMD create dkg --output-dir=/charon-docker-compose --num-validators=2 --threshold=${T} --operator-enrs=${ENRS} --dkg-algorithm=frost
```
[the remark this answers is not in the document]
Comment (corverroos) [reply configure_dkg.sh:21]: I think it is nice to test a cluster with more than 1 DV. Why only 1?

File: Makefile [hunk 3]
```diff
@@ -70,3 +71,10 @@ disable-simnet:
 	@if [ "$(beacon_node_endpoint)" == "" ]; then echo "Please provide beacon_node_endpoint variable: make beacon_node_endpoint=<value> disable-simnet" && exit 1; fi
 	@if [ ! -f beaconnode.env.old ]; then mv beaconnode.env beaconnode.env.old; fi
 	@echo "CHARON_BEACON_NODE_ENDPOINT=$(beacon_node_endpoint)" > beaconnode.env
+
+.PHONY: dkg
+dkg:
+	@if [ ! -f charon ]; then echo "Pegged version doesn't support DKG yet, please run: make build-local" && exit 1; fi
```
[the remark this answers is not in the document]
Comment (dB2510) [reply Makefile:77]: cool 👍

**Pull request merged**